src/SamsungMdc.cs

```diff
@@ -26,10 +26,10 @@
 
 namespace PepperDashPluginSamsungMdcDisplay
 {
-    public class SamsungMdcDisplayController : TwoWayDisplayBase, IBasicVolumeWithFeedback, ICommunicationMonitor,
+    public class SamsungMdcDisplayController : PepperDash.Essentials.Devices.Common.Displays.TwoWayDisplayBase, IBasicVolumeWithFeedback, ICommunicationMonitor,

```

Comment:
This should be added as a `using` statement next time, rather than fully qualifying the type.